extensions/amp-story-interactive/validator-amp-story-interactive.protoascii

```diff
@@ -281,3 +347,47 @@ attr_lists: {
     }
   }
 }
+attr_lists: {
```

Comment:
should these be `value_url` to enforce a protocol such as `https`?

Reply:
Yes, let's match `amp-story-page-attachment[cta-image]`:
```
  attrs: {
    name: "cta-image"
    value_url: {
      protocol: "http"
      protocol: "https"
    }
  }
```

Reply:
@gmajoulet should it allow relative urls or only absolute?